ceph.spec.in

```diff
@@ -196,10 +196,10 @@ BuildRequires:	python%{_python_buildid}-requests
 BuildRequires:	python%{_python_buildid}-six
 BuildRequires:	python%{_python_buildid}-tox
 BuildRequires:	python%{_python_buildid}-virtualenv
-%if 0%{?fedora} || 0%{?rhel}
+%if 0%{?rhel}
 BuildRequires:  pyOpenSSL%{_python_buildid}
 %endif
-%if 0%{?suse_version}
+%if 0%{?fedora} || 0%{?suse_version}
```

Comment:
#26520 also takes RHEL8 into consideration. so i'm inclined to have @ktdreyer 's fix.

Reply:
sounds good to me!

Reply:
shall we close this?

Reply:
yeah, please!